datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -58,9 +84,9 @@ def create_codeowners_map():
 def codeowners(ctx):
     """Validate that every integration has an entry in the `CODEOWNERS` file."""
 
-    has_failed = False
     is_core_check = ctx.obj['repo_choice'] == 'core'
 
+    has_failed = False
```

Comment:
lets keep these lines as they were. For a second I thought you were still using the boolean flag, but then I realized it was the old code that still relied on it.

Reply:
Which lines do you mean ? is it the `has_failed` assignment or all the lines ?

Reply:
just the `has_failed` assignment

Reply:
Sure, done in: 280e802ec2